WWDC/Realm+Combine.swift

```diff
@@ -11,18 +11,30 @@ import RealmSwift
 
 extension RealmSubscribable where Self: Object {
     func valuePublisher(share: Bool = true, includeInitialValue: Bool = true, keyPaths: [String]? = nil) -> some Publisher<Self, Error> {
-        var initialValue: some Publisher<Self, Error> { Just(self).setFailureType(to: Error.self) }
         var valuePublisher: some Publisher<Self, Error> { RealmSwift.valuePublisher(self, keyPaths: keyPaths) }
 
+        // by using `.multicast(subject: CurrentValueSubject(self))` we are sharing the upstream publisher
+        // and also replaying the most recent value to new subscribers which is useful when
+        // subscriptions come in at slightly different times
         switch (share, includeInitialValue) {
         case (true, true):
-            return Publishers.Concatenate(prefix: initialValue, suffix: valuePublisher.share()).eraseToAnyPublisher()
+            return valuePublisher
+                .prepend(self)
+                .multicast(subject: CurrentValueSubject(self))
```

Comment:
I don’t think I need both prepend and the current value subject.

CurrentValueSubject always emits its most recent value when subscribing to it. This has the effect of being a multicast (one realm subscription) that will replay its latest value to subscribers. 

But I really need to double check that. Ultimately this is just a more nuanced version of the share() operator.